frontend/src/metabase/query_builder/components/notebook/NotebookSteps/NotebookSteps.tsx

```diff
@@ -92,7 +89,7 @@ function NotebookSteps({
   }
 
   return (
-    <Container className={className}>
+    <>
```

Comment:
The `Container` styled component was simply adding a top margin (now handled in the notebook), and the `className` prop was not used at all.